app/controllers/partners/base_controller.rb

```diff
@@ -10,5 +10,32 @@ class BaseController < ApplicationController
     def redirect_to_root
       redirect_to root_path
     end
+
+    helper_method :current_partner
+    def current_partner
+      current_partner_user.partner
+    end
```

Comment:
I'll make an issue to start utilizing this method instead of calling `current_partner_user` :)